### DISCUSSION TITLE
Output $tag->url tidak sesuai

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#268] [Q&A] [answered]
Kang, ane mau tampilin Tags, tetapi url output dari `$tag->url` nya seperti ini `http://localhost/hero/C:/xampp2/htdocs/hero/lot/tag/dokter-operasional`, bagian mana yang harus diperbaiki kang

```
<?php $tags = Pages::from(LOT . D . 'tag', true)->sort([-1, 'time'])->chunk(5,0);?>
<?php foreach($tags as $tag): ?>
    <a href="<?=$tag->url;?>"><?=$tag->title;?>~ ID: <?=$tag->id;?></a>
    <?php endforeach; ?>
```

![image](https://github.com/mecha-cms/mecha/assets/96925342/4e14719c-eaf6-4bc5-969c-ef964b3a2df4)

**Comment by yuliakbard** [comment, marked as answer]
sepertinya sudah fix.

```
<?php $tags = Tags::from(LOT . D . 'tag', true)->sort([-1, 'time'])->chunk(5,0);?>
<?php foreach($tags as $tag): ?>
    <a href="<?=$tag->link;?>"><?=$tag->title;?>~ ID: <?=$tag->id;?></a>
    <?php endforeach; ?>
```

**Comment by taufik-nurrohman** [comment]
Tag sementara ini tidak punya properti `url` karena memang tidak ada laman khusus untuk menampilkan bekas tag tersebut.
